pkgs/development/python-modules/shiboken2/default.nix

```
@@ -26,6 +26,9 @@ stdenv.mkDerivation {
   dontWrapQtApps = true;
 
   postInstall = ''
+    cd ../../..
+    python setup.py egg_info --build-type=shiboken2
```

```suggestion
    ${python.interpreter} setup.py egg_info --build-type=shiboken2
```
Can't we use the standard pythonPackages tools instead?

---

I'm not sure what you mean. Is there an alternative way to build the egg info?

---

no, we prefer to use `${python.interpreter}` so that interpreters like `pypy` can be used. Hard coding `python` pretty much vendors you into only using cpython. Which is what you want 99% of the time, but hey, why encode assumptions?

---

Got it, thanks. Switched this to `${python.interpreter}`.